app/assets/stylesheets/common/admin/admin_base.scss

```diff
@@ -260,6 +260,9 @@ $mobile-breakpoint: 700px;
       margin-top: 1em;
     }
   }
+  .warning {
+    color: #e45735;
```

Comment:
This should be based on our SCSS variables for theme support. I'd also suggest making it `p.warning` in case the generic `warning` class is used elsewhere.